query_optimizer/logical/Sample.hpp

```diff
@@ -0,0 +1,133 @@
+/**
+ *   Copyright 2016, Quickstep Research Group, Computer Sciences Department,
+ *   University of Wisconsin—Madison.
```

Comment:
The existing files might still have leftover style issues, which should be fixed in a separate PR.

For new files, we should stick to the style guidance.